config/jobs/kubernetes-sigs/aws-ebs-csi-driver/OWNERS

```diff
@@ -7,10 +7,13 @@ reviewers:
 - leakingtapan
 - torredil
 - wongma7
+- AndrewSirenko
+- elijahlq
```

Comment:
Please keep the list alphabetically sorted (ditto for the other addition below)

Also, `elijahlq` is not a GitHub username - https://github.com/elijahlq